backend/Models/Coordinates.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Internships\Models;
+
+use JsonSerializable;
+
+class Coordinates implements JsonSerializable
+{
+    protected float $latitude;
+    protected float $longitude;
+
+    public function __construct(string $coordinates)
+    {
+        $arrCoordinates = explode(",", preg_replace("/\s+/", "", $coordinates));
+        $this->latitude = floatval($arrCoordinates[0]);
+        $this->longitude = floatval($arrCoordinates[1]);
+        $this->normalize();
+    }
+
+    public function getLatitude(): float
+    {
+        return $this->latitude;
+    }
+
+    public function getLongitude(): float
+    {
+        return $this->longitude;
+    }
+
+    public function jsonSerialize(): array
+    {
+        return [
+            "latitude" => $this->latitude,
+            "longitude" => $this->longitude,
+        ];
+    }
+
+    protected function normalize(): void
+    {
+        $this->latitude = floatval(number_format($this->latitude, 6));
```

Comment:
Are you sure that you want to use float casting here? http://sandbox.onlinephpfunctions.com/code/b191152d93c766b6c686c3833a713a6041abe017

Reply:
I see it cuts zeroes, but I'm not sure if that's an issue. Most applications don't display them and they aren't needed to be passed into MapBox.

Though I'm not sure if there's rounding issue, so I'll investigate that further.